src/OpenFga.Sdk/Client/Model/ClientListObjectsRequest.cs

```diff
@@ -0,0 +1,111 @@
+using System.Runtime.Serialization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace OpenFga.Sdk.Client.Model;
+
+internal interface IClientListObjectsRequest {
+    public string User { get; set; }
+    public string Relation { get; set; }
+    public string Type { get; set; }
+    public List<TupleKey>? ContextualTuples { get; set; }
+}
+
+/// <summary>
+///     ClientListObjectsRequest
+/// </summary>
+public class ClientListObjectsRequest : IClientListObjectsRequest, IEquatable<ClientListObjectsRequest>,
+    IValidatableObject {
+    /// <summary>
+    ///     Gets or Sets User
+    /// </summary>
+    [DataMember(Name = "user", EmitDefaultValue = false)]
+    [JsonPropertyName("user")]
+    public string User { get; set; }
+
+    /// <summary>
+    ///     Gets or Sets Relation
+    /// </summary>
+    [DataMember(Name = "relation", EmitDefaultValue = false)]
+    [JsonPropertyName("relation")]
+    public string Relation { get; set; }
+
+    /// <summary>
+    ///     Gets or Sets Type
+    /// </summary>
+    [DataMember(Name = "type", EmitDefaultValue = false)]
+    [JsonPropertyName("type")]
+    public string Type { get; set; }
+
+    /// <summary>
+    ///     Gets or Sets Contextual Tuples
+    /// </summary>
+    [DataMember(Name = "contextual_tuples", EmitDefaultValue = false)]
+    [JsonPropertyName("contextual_tuples")]
+    public List<TupleKey>? ContextualTuples { get; set; }
+
+    public bool Equals(ClientListObjectsRequest input) {
+        if (input == null) {
+            return false;
+        }
+
+        return
+            (
+                Type == input.Type ||
+                (Type != null &&
+                 Type.Equals(input.Type))
+            ) &&
+            (
+                Relation == input.Relation ||
+                (Relation != null &&
+                 Relation.Equals(input.Relation))
+            ) &&
+            (
+                User == input.User ||
+                (User != null &&
+                 User.Equals(input.User))
+            ) &&
+            (
+                ContextualTuples == input.ContextualTuples ||
+                (ContextualTuples != null &&
+                 ContextualTuples.Equals(input.ContextualTuples))
+            );
+    }
+
+    public IEnumerable<ValidationResult> Validate(ValidationContext validationContext) {
+        yield break;
+    }
+
+    public virtual string ToJson() => JsonSerializer.Serialize(this);
+
+    public static ClientListObjectsRequest FromJson(string jsonString) =>
+        JsonSerializer.Deserialize<ClientListObjectsRequest>(jsonString) ?? throw new InvalidOperationException();
+
+    public override bool Equals(object input) => Equals(input as ClientListObjectsRequest);
+
+    public override int GetHashCode() {
+        unchecked // Overflow is fine, just wrap
+        {
+            var hashCode = 9661;
+            if (Type != null) {
+                hashCode = (hashCode * 9923) + Type.GetHashCode();
```

Comment:
can we set 9961/9923 magic number to be a constant?

Reply:
They are constants defined in the generator: https://github.com/openfga/sdk-generator/blob/main/config/clients/dotnet/config.overrides.json#L36-L37